packages/application-shell/src/components/quick-access/quick-access.spec.js

```diff
@@ -3,7 +3,12 @@ import {
   GRAPHQL_TARGETS,
   MC_API_PROXY_TARGETS,
 } from '@commercetools-frontend/constants';
-import { renderAppWithRedux, fireEvent, wait } from '../../test-utils';
+import {
+  renderAppWithRedux,
+  fireEvent,
+  wait as waitFor,
+  waitForElementToBeRemoved,
```

Comment:
Btw, this is not used apparently.

Reply:
Yeah I noticed ESLint being more loose on app-kit. Maybe something to change.